### PR TITLE
Update the image stream resource

### PR DESCRIPTION
**Pull request by duyanyan**

### CODE REVIEW

**Review by myakove — APPROVED** [review]

**Comment by myakove** [comment]
!verified

**Comment by myakove** [comment]
!verified

**Comment by myakove** [comment]
!verified

**Comment by myakove** [comment]
@duyanyan it was changed to !verified

File: ocp_resources/image_stream.py [hunk 1]
```diff
@@ -7,3 +8,51 @@ class ImageStream(NamespacedResource):
...
+        self.tag = tag
+
+    def to_dict(self):
+        res = super().to_dict()
+        if self.yaml_file:
+            return res
+        res.update(
+            {
+                "spec": {
+                    "lookupPolicy": {"local": False},
```
Comment (rnetser) [inline ocp_resources/image_stream.py:47]: lookupPolicy.local should be configurble; you can set the default to False

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/image_stream.py (hunk 1 above)
Comment (duyanyan) [reply ocp_resources/image_stream.py]: Fixed

**Comment by duyanyan** [comment]
!verified

**Review by myakove — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**